lib/diagnostic.js

```diff
@@ -6,29 +6,69 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (string) {
+
+let words = string.replace(/\s+/);
+for (let i = 0; i < words.length; i++) {
+  let words = words[i].replace(/\W+/, '');
+  normalizedWords[i] = words.toUpperCase();
+}
+};
+
 };
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+
+const uniqueWords = function (normalizedWords) {
+  let uniqueWordsAsKeys = {};
+  for (let i = 0; i < normalizedWords.length; i++) {
+  let word = normalizedWords[i];
+  uniqueWordsAsKeys[word] = true;
+  }};
+
+let uniqueWords = [];
```

Comment:
Same issues here as above. Your function is called `uniqueWords`, which is a `const`, and here you try to define another `uniqueWords` as a let. Please see the solution branch.